api/services/AuthorizationPolicy/utils.js

```diff
@@ -8,3 +8,5 @@ export const isUser = (viewer) =>
 export const isSelf = (user, viewer) => user?.id == viewer?.id;
 
 export const isOrganiser = (event, viewer) => event?.organiser_id == viewer?.id;
+
+export const isPublicEvent = (event) => event?.is_public === true;
```

Comment:
remove this

Reply:
but we later on , when we will have not logged in user we will open the events/id_or_slug in the basis of is_public

Reply:
okay fine you want me to check value.is_public directly in the policy

Reply:
yes